tests/installation/user_settings.pm

```diff
@@ -38,11 +39,16 @@ sub run {
     my $max_tries = 4;
     my $retry     = 0;
     do {
-        $self->enter_userinfo(retry => $retry);
+        if ($retry == 0) {
+            $self->enter_userinfo();
+        } else {
+            $self->enter_userinfo(max_interval => utils::VERY_SLOW_TYPING_SPEED);
```

Comment:
I'd go for something like this, but the inner me says: ]Explicit is better than implicit](https://www.python.org/dev/peps/pep-0020/)
```suggestion
            $self->enter_userinfo(max_interval => ($retry)?  utils::VERY_SLOW_TYPING_SPEED : undef );
```

Reply:
Exactly. there is no gain in doing a one-liner.

Reply:
hey, this is perl, meaning … probably we would need to follow the opposite ;P

Reply:
More than one way to do it. I choose python way.